azure-mgmt-batchai/src/main/java/com/microsoft/azure/management/batchai/BatchAIFileServer.java

```diff
@@ -108,24 +116,62 @@
             WithUserName withVMSize(String vmSize);
         }
 
+        /**
+         * This stage of a Batch AI file server definition allows to specify administrator account name.
+         */
         interface WithUserName {
+            /**
+             * @param userName the name of the administrator account
+             * @return the next stage of the definition
+             */
             WithUserCredentials withUserName(String userName);
         }
 
+        /**
+         * This stage of a Batch AI file server definition allows to specify user credentials.
+         */
         interface WithUserCredentials {
+            /**
+             * @param password admin user Password (linux only)
+             * @return the next stage of the definition
+             */
             WithCreate withPassword(String password);
 
+            /**
+             * @param sshPublicKey SSH public keys used to authenticate with linux based VMs
+             * @return the next stage of the definition
+             */
             WithCreate withSshPublicKey(String sshPublicKey);
         }
 
+        /**
+         * Defines subnet for the cluster.
+         */
+        interface WithSubnet {
+            /**
+             * @param subnetId identifier of the subnet
+             * @return the next stage of the definition
+             */
+            WithCreate withSubnet(String subnetId);
+
+            /**
+             * @param networkId identifier of the network
+             * @param subnetName subnet name
+             * @return the next stage of the definition
+             */
+            WithCreate withSubnet(String networkId, String subnetName);
+        }
```

Comment:
Please add comments for all methods in the above interfaces

Reply:
Done.